src/namespace/privilegednamespace.js

```diff
@@ -138,6 +139,35 @@ export class PrivilegedNamespace extends CustomNamespace {
         callback(debate.debateID);
     };
 
+    /**
+     * Return the true if the debate was closed correctly false otherwise in the callback function
+     */
+    closeDebate = (socket) => async (aIdDiscussion, callback) => {
+        logger.debug(`Close debate requested from ${socket.username}`);
+
+        if (!(callback instanceof Function)) {
+            logger.debug(`callback is not a function.`);
+            return;
+        }
+
+        // Get the debate with the desired id
+        let debate = this.getActiveDebate(aIdDiscussion);
+        logger.debug(`Debate given ${debate}`);
+        // If the debate does not exist it cannot be closed
+        if(debate == null){
+            callback(false);
```

Comment:
Add a ` logger.debug(...);` instruction here with a text similar to "no active debate with the id ${aIdDiscussion} were found".

Reply:
Change done in the latest commit